spot_rl_experiments/spot_rl/models/od_aria.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+import time
+from typing import Optional
+
+import numpy as np
+from owlvit import OwlVit
```

Comment:
I didnt see details on OwlVIT in install/setup instructions. Should we add?

Reply:
OWL-ViT being used here is from spot-rl. I will add in the readme that if needed, user should make sure to install spot-rl pkg.